webofneeds/won-owner-webapp/src/main/webapp/app/actions/actions.js

```diff
@@ -573,3 +576,16 @@ export function startTicking() {
       60000
     );
 }
+
+function toggleDebugMode() {
+  return dispatch => {
+    won.debugmode = !won.debugmode;
+    dispatch({
+      type: actionTypes.view.toggleDebugMode,
+      payload: won.debugmode,
+    });
+    const text = won.debugmode ? "Debugmode On" : "Debugmode Off";
+    // this.props.toastPush(Immutable.fromJS({ text }));
+    dispatch(actionCreators.toasts__push(Immutable.fromJS({ text })));
```

Comment:
imstead of pushing a toast with a specific message we can use the actionType from the line above to display a toast according to the state within the toast-reducer itself, that way we only have a single dispatch instead of two

Reply:
done :)